SecondStageUpdater/Program.cs

```diff
@@ -34,6 +34,13 @@ internal sealed class Program
     private static async Task Main(string[] args)
     {
         defaultColor = Console.ForegroundColor;
+#if DEBUG
+        args = new[]
+        {
+            "clientogl.dll",
+            "\"C:\\Users\\RansAckeR\\source\\repos\\xna-cncnet-client\\DXMainClient\\bin\\Debug\\YR\\\""
+        };
+#endif

```

Comment:
Unintended?

Reply:
If not unintended, can use `app.config` instead?

Reply:
Unintended indeed, however if needed one can use \Properties\launchSettings.json file with commandLineArgs.